### PR TITLE
Add runtime environment output to fix building the release container

### PR DESCRIPTION
**Pull request by cwharris**
## Description

Fixes the release container build.

## By Submitting this PR I confirm:
- I am familiar with the [Contributing Guidelines](https://github.com/nv-morpheus/Morpheus/blob/main/docs/source/developer_guide/contributing.md).
- When the PR is ready for review, new or existing tests cover these changes.
- When the PR is ready for review, the documentation is up to date with these changes.

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -176,13 +176,13 @@ ARG CUDA_MINOR_VER
 ARG CONDA_CHANNEL_ALIAS
 
 # Copy the development dependencies file
-COPY ${MORPHEUS_ROOT_HOST}/docker/conda/environments/cuda${CUDA_MAJOR_VER}.${CUDA_MINOR_VER}_dev.yml ./docker/conda/environments/
+COPY ${MORPHEUS_ROOT_HOST}/docker/conda/environments/dev_cuda-${CUDA_MAJOR_VER}${CUDA_MINOR_VER}_arch-x86_64.yml ./docker/conda/environments/dev.yaml
```
Comment (mdemoret-nv) [inline docker/Dockerfile:179]: Shouldnt the environments under `docker/conda` be deleted by now and we should be using the ones in `conda/environments`?

File: docker/build_container.sh [hunk 2]
```diff
@@ -28,9 +28,9 @@ DOCKER_TARGET=${DOCKER_TARGET:-"runtime"}
 DOCKER_EXTRA_ARGS=${DOCKER_EXTRA_ARGS:-""}
 
 # Build args
-CUDA_MAJOR_VER=${CUDA_MAJOR_VER:-11}
-CUDA_MINOR_VER=${CUDA_MINOR_VER:-8}
-CUDA_REV_VER=${CUDA_REV_VER:-0}
+CUDA_MAJOR_VER=${CUDA_MAJOR_VER:-12}
```
Comment (mdemoret-nv) [inline docker/build_container.sh:31]: I think we can also delete the references to `RAPIDS_VERSION` and `TENSORRT_VERSION` in this file.

**Review by mdemoret-nv — CHANGES_REQUESTED** [review]
Just want to confirm that we are using the new environments in `conda/environments` and all references to `docker/conda` has been removed.

File: docker/Dockerfile (hunk 1 above)
Comment (cwharris) [reply docker/Dockerfile:179]: That's correct. They have been deleted, which was the primary reason this was failing to build.

**Comment by cwharris** [comment]
Addressed the feedback and opened https://github.com/nv-morpheus/Morpheus/issues/1497. We'll need to work with the data science team to make sure we get this updated properly, so suggest we go ahead and merge this PR.

**Review by mdemoret-nv — APPROVED** [review]

**Pull request merged**